app/lib/constants.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     PERIOD_END: 'PeriodEnd',
     SUPPLIER: 'Supplier',
   },
-  OUTPUT_DIR: 'site/json',
+  OUTPUT_DIR: 'output',
+  CONTAINER_NAME: process.env.CONTAINER_NAME || 'etl-output'
```

Comment:
To avoid conflicts we took the approach of having separate blob storage per area, but with a consistent container name, so there is blob storage for pharmacy data and for GP data which both have 'etl-output' as the container name.

If there's still concern over two ETLs writing to the same location, additional blob storage could be created just for the POMI data.